content/references/protocol-reference/transactions/transaction-types/clawback.md

```diff
@@ -0,0 +1,64 @@
+---
+html: clawback.html
+parent: transaction-types.html
+blurb: Claw back an issued currency transfer between accounts in a trust set.
+labels:
+  - Tokens
+status: not_enabled
+---
+# Clawback
+
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/ripple/app/tx/impl/Clawback.cpp "Source")
+
+<embed src="/snippets/_clawback-disclaimer.md" />
```

Comment:
Same as above re:snippet

Reply:
Thank you, Rome, for your thorough review.